### PR TITLE
Disable radio buttons, and optimize for mobile.

### PR DESCRIPTION
**Pull request by DrewBradley**
I adjusted the media queries to be more responsive. I would like to find a way to use the gradient as the background on the mobile version as well.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +30,7 @@ <h2 class="subtitle">Which type of message?</h2>
         <button class="personal-button">Customize</button>
         <button class="clear-message hidden">Clear Message</button>
       </form>
+      <!-- MESSAGE FORM -->
```
Comment (mike-duke) [inline index.html:33]: If you're going to create section comments like this, it would make sense to do it throughout the document.

File: main.js [hunk 2]
```diff
@@ -49,13 +47,14 @@ function returnMantra() {
...
-
 function returnMessage() {
     event.preventDefault();
     image.classList.add('hidden');
     messageText.classList.remove('hidden');
+    affirmationButton.disabled = true;
+    mantraButton.disabled = true;
     if (!affirmationButton.checked && !mantraButton.checked && !messageInput.value){
-        alert('Please select a message type!')
+        // alert('Please select a message type!')
```
Comment (mike-duke) [inline main.js:57]: As this is a personal project, it doesn't matter as much, but out in the wild you will want to remove all commented out code before opening a PR that someone else will check.

File: styles.css [hunk 3]
```diff
@@ -113,9 +113,52 @@ button {
...
+        border-radius: 5px;
+        border: none;
+        color: white;
+        padding: 5%;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 1em;
+        margin: 5% 2%;
+        cursor: pointer;
```
Comment (mike-duke) [inline styles.css:152]: Are you changing all of these properties?

File: styles.css [hunk 4]
```diff
@@ -113,9 +113,52 @@ button {
     width: auto;
 }
 
-@media screen and (max-width: 300px) {
+@media screen and (max-width: 600px) {
```
Comment (mike-duke) [inline styles.css:116]: make sure that this is the actual size that you want the screen to change and not an arbitrary value.

**Review by mike-duke — APPROVED** [review]
Looks good overall. Comments in the code are mostly for reflection.

File: main.js (hunk 2 above)
Comment (DrewBradley) [reply main.js:57]: Thank you! That is incredibly helpful!
No commented code in PRs. Thank you!

File: styles.css (hunk 4 above)
Comment (DrewBradley) [reply styles.css:116]: Yes! I was messing with it when my media queries weren't working. Thank you for trouble shooting that with me. Meta tag!

File: styles.css (hunk 3 above)
Comment (DrewBradley) [reply styles.css:152]: I believe so. Is there a way I need to shorten this?

**Pull request merged**